### PR TITLE
Datatalk podcast plug with RAG Eval

### PR DESCRIPTION
**Pull request by atarora**
Added DeepRec Event info too.

### CODE REVIEW

File: qdrant-landing/content/blog/datatalk-club-podcast-plug.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Navigating challenges and innovations in search technologies
```
Comment (mjang) [inline qdrant-landing/content/blog/datatalk-club-podcast-plug.md:1]: Adding header info. Please modify as you'd like:
```suggestion
title: "Navigating challenges and innovations in search technologies"
draft: false
slug: navigating-challenges-innovations
short_description: Podcast on search and LLM with Datatalks
description: <substitute>
date: <substitute publication date>
author: Atita Arora
featured: false
tags:
  - podcast
  - search
  - blog
  - retrieval-augmented generation
  - large language models
---

## Navigating challenges and innovations in search technologies
```

File: qdrant-landing/content/blog/datatalk-club-podcast-plug.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Navigating challenges and innovations in search technologies"
+draft: false
+slug: navigating-challenges-innovations
+short_description: Podcast on search and LLM with Datatalk.club
+description: Podcast on search and LLM with Datatalk.club
+date: 12.01.2024
```
Comment (generall) [inline qdrant-landing/content/blog/datatalk-club-podcast-plug.md:7]: the date should be specified in the format like this: `2022-09-26T15:39:53.751Z`

**Review by generall — CHANGES_REQUESTED** [review]
Please specify at least `preview_image`.

**Comment by atarora** [comment]
> Please specify at least `preview_image`.

Done !

File: qdrant-landing/content/blog/datatalk-club-podcast-plug.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Navigating challenges and innovations in search technologies"
+draft: false
+slug: navigating-challenges-innovations
+short_description: Podcast on search and LLM with Datatalk.club
+description: Podcast on search and LLM with Datatalk.club
+preview_image: /blog/navigating-challenges-innovations/preview/preview.png
+date: 12.01.2024T15:39:53.751Z
```
Comment (mjang) [inline qdrant-landing/content/blog/datatalk-club-podcast-plug.md:8]: Hugo (doc tool) expects a different date format. This makes the article link appear at https://qdrant.tech/blog/

```suggestion
date: 2024-01-12T15:39:53.751Z
```

**Review by mjang — APPROVED** [review]
We discussed the draft in raw form before @atarora submitted this PR

**Pull request merged**